pages/linux/brew.md

```diff
@@ -14,15 +14,15 @@
 
 `brew list`
 
-- Update an installed formula (passing no `{{formula}}` updates all installed formulae):
+- Update an installed formula (if no formula name is given, updates all installed formulae):
```

Comment:
Perhaps it would read better if it said `all installed formulae are updated` instead of `updates all installed formulae`?

Reply:
Agreed.